docs: update cors documentation (#1241)
elastic/apm-agent-rum-js: docs/set-up.asciidoc

Access-Control-Allow-Origin: [request-origin]
----

> If you enable the <<send-credentials, sendCredentials>> configuration option you will need to make sure that your proxy's response include the header `Access-Control-Allow-Origin` with the page's origin as a value and that the following header is also included:

Reviewer: ```suggestion
If you enable the <<send-credentials, `sendCredentials`>> configuration option, your proxy's response must include the header `Access-Control-Allow-Origin` with the page's origin as a value, and the following header:
```

Author: thanks @bmorelli25!
